libs/game/tilemap.ts

```diff
@@ -257,5 +255,24 @@ namespace tiles {
                 index
             );
         }
+
+        public isOnWall(s: Sprite) {
+            const hbox = s._hitbox
+
+            const left = Fx.toIntShifted(Fx.add(hbox.left, Fx.oneFx8), this.scale);
```

Comment:
Why do you add 1 to the left and top?

Reply:
Ahh, that was to fix an off by one bug, but I ended up fixing the issue causing it / rendering the offset moot. Removed~